packages/subgraph/src/plugin/utils.ts

```diff
@@ -61,9 +65,23 @@ function createTokenVotingPlugin(plugin: Address, daoId: string): void {
     packageEntity.minDuration = minDuration.reverted ? null : minDuration.value;
 
     if (!token.reverted) {
-      let contract = fetchERC20(token.value);
-      if (contract) {
-        packageEntity.token = contract.id;
+      let tokenAddress = token.value;
+      if (supportsERC20Wrapped(tokenAddress)) {
+        let contract = fetchWrappedERC20(tokenAddress);
+        if (!contract) {
+          return;
+        }
+        if (contract) {
```

Comment:
if is not necessary. because there is a contract otherwise line 72 would have already aborted this code path

Reply:
amended